docs/md/usage-retry.md

```diff
@@ -1,3 +1,28 @@
+# Retry Workflows in Terra
+
+WFL has `/retry` endpoint
+that selects workflows
+by their completion status
+and re-submits them.
+
+The following `curl` shell command
+finds the workflows with the status `"Failed"`
+in the workload with `$UUID`
+and resubmits them for processing.
+
+```bash
+export WFL=https://gotc-prod-wfl.gotc-prod.broadinstitute.org/api/v1/workload
+export AUTH="Authorization: Bearer $(gcloud auth print-access-token)"
+export UUID=0d307eb3-2b8e-419c-b687-8c08c84e2a0c # workload UUID
+
+curl -X POST -H "$AUTH" $WFL/$UUID/retry --data '{"status":"Failed"}' | jq
+```
+
+The `/retry` endpoint is not yet implemented.
+and returns a `501` HTTP failure status.
+Until it is supported,
+see the method below.
```

Comment:
When the code is released, it will. Could you write this as if it is correct at the time of release to the best of our knowledge?

Reply:
OK. I'll give it a shot.